tests/test_volumetricimagingextractor.py

```diff
@@ -9,8 +9,8 @@
 
 @pytest.fixture(scope="module", params=[1, 2])
 def imaging_extractors(request):
-    num_channels = request.param
-    return [generate_dummy_imaging_extractor(num_channels=num_channels, num_samples=num_samples) for _ in range(3)]
+    # num_channels parameter is deprecated and removed - all extractors are single channel now
```

Comment:
If we are removing `num_channels`, this test fixture probably should not have an unused parameter that references `num_channels`.